File: C++/lib/Calculator/Calculator.cpp
#include <stdio.h>
#include <conio.h>
#include <iomanip>
#include <iostream>
using namespace std;

int add(){
        cout << "\n\nEnter '0' to End addition";
        double var1 = 0 , var2 = 0;
        cout << "\nEnter any number: ";
        cin >> var1;
        if(var1 == 0){
            exit(0);
        }
    do{
        cout << fixed;
        cout << setprecision(2);
        cout << "Enter any number: ";
        cin >> var2;
        if(var2 == 0){
            cout << endl << var1;
            break;
        }
        var1 += var2;
    }while (1);
    getch();
    return 0;
}
int sub(){
        cout << "\n\nEnter '0' to End Subtraction";
        double var1 = 0 , var2 = 0;
        cout << "\nEnter any number: ";
        cin >> var1;
        if(var1 == 0){
            exit(0);
        }
    do{
        cout << fixed;
        cout << setprecision(2);
        cout << "Enter any number: ";
        cin >> var2;
        if(var2 == 0){
            cout << endl << var1;
            break;
        }
        var1 -= var2;
    }while (1);
    getch();
    return 0;
}
int div(){
        cout << "\n\nEnter '0' to End Division";
        double var1 = 0 , var2 = 0;
        cout << "\nEnter any number: ";
        cin >> var1;
        if(var1 == 0){
            exit(0);
        }
    do{
        cout << fixed;
        cout << setprecision(2);
        cout << "Enter any number: ";
        cin >> var2;
        if(var2 == 0){
            cout << endl << var1;
            break;
        }
        var1 /= var2;
    }while (1);
    getch();
    return 0;
}
int multi(){
        cout << "\n\nEnter '0' to End Multiplication";
        double var1 = 0 , var2 = 0;
        cout << "\nEnter any number: ";
        cin >> var1;
        if(var1 == 0){
            exit(0);
        }
    do{
        cout << fixed;
        cout << setprecision(2);
        cout << "Enter any number: ";
        cin >> var2;
        if(var2 == 0){
            cout << endl << var1;
            break;
        }
        var1 *= var2;
    }while (1);
    getch();
    return 0;
}

int main(){
	char choice;
    do{
        system ("color B");
        printf("\nMenu");
        printf ("\n\n0. Exit                    press '0' to Exit");
        printf("\n+. Addition                press '+' for Addition");
        printf("\n-. Subtraction             press '-' for Subtraction");
        printf("\n/. Division                press '/' for Division");
        printf("\nx. Multiplication          press 'x' for Multiplication");

        printf("\n\nEnter your choise: ");
        scanf ("%c" , &choice);

            if (choice == '+'){
                add();
            }else if (choice == '-'){
                sub();
            }else if (choice == '/'){
                div();
            }else if(choice == 'x'){
                multi();
            }else if (choice == '0'){
                exit(0);
            }
            system("cls");
    }while (1);
    return 0;
}

File: Python/lib/calculator/Calculator.py
from msvcrt import getch
from os import system
from lib.calculator.addition import add
from lib.calculator.subtraction import sub
from lib.calculator.multiiplication import multi
from lib.calculator.division import div


def Calculator():
    while True:
        system('cls')
        print('0. Exit')
        print('1. Addition')
        print('2. Subtraction')
        print('3. Multiplication')
        print('4. Division\n')

        choice = int(input('Enter your Choice: '))

        if choice == 1:
            lst_add = [float(inp_add) for inp_add in input('Enter Number: ').split(' + ')]
            print('')
            add(lst_add)
            getch()
        elif choice == 2:
            lst_sub = [float(inp_sub) for inp_sub in input('Enter Number: ').split(' - ')]
            print('')
            print(sub(lst_sub))
            getch()
        elif choice == 3:
            lst_multi = [float(inp_multi) for inp_multi in input('Enter Number: ').split(' x ')]
            print('')
            multi(lst_multi)
            getch()
        elif choice == 4:
            lst_div = [float(inp_div) for inp_div in input('Enter Number: ').split(' / ')]
            print('')
            print(div(lst_div))
            getch()
        elif choice == 0:
            print('Thank you')
            getch()
            quit()
        else:
            print('')
            print('Error!')
            getch()
        print('')


Calculator()
